### DISCUSSION TITLE
I want to define a default country and cancel country selection

### DISCUSSION THREAD

**Discussion by ysnbyzli** [AstrOOnauta/react-native-international-phone-number#4] [Q&A] [answered]
Hello,

I want to set only one country as default and I don't want the user to change this country, how can I do that? I would be glad if you help.

Thanks.

**Comment by AstrOOnauta** [comment, marked as answer]
Hey @ysnbyzli !

About your question, nice point!
The version 0.4.10 or earlier does not support this feature, but I released a next version with new feat to disable the country modal.
Please update the lib version to v0.4.11 and use the following code to resolve your issue:

```
import { PhoneInput, getCountryByCca2 } from 'react-native-international-phone-number';
...
 const [selectedCountry, setSelectedCountry] = useState(getCountryByCca2("BR");
...
 <PhoneInput
    ...
    modalDisabled   <--- Add This Props
  />

```

OBS.: Referenced in this new point in the docs:
https://github.com/AstrOOnauta/react-native-international-phone-number#country-modal-disabled-mode

🚀AstrOOnauta 🚀

**Reply by ysnbyzli** [reply]
Thanks for your descriptive answer 🙏🏻

@AstrOOnauta

**Reply by Zahid40** [reply]
very helpfull
